### ISSUE TITLE
fill in the missing values of the sparse depth maps of kitti dataset

### ISSUE THREAD

**Issue by YifeiAI** [xanderchf/MonoDepth-FPN-PyTorch#1]
Hi,

Thanks for posting your code!

I have a question for the depth maps in kitti dataset. You mentioned that before training you use the NYu tool box to fill in the sparse depth maps of kitti dataset. I am wondering:

1. Are the sparse depth image from the official website under the section "depth completion/depth prediction" with 14G?

2. Can you tell more specifically how to use the tool box to fill in the sparse depth map? And how accurate is it to use the filled-in depth map as ground truth label.

Thank you very much!

**Comment by xanderchf** [comment]
Hello,

1. Yes, that is correct.
2. You can go to the github link given in the readme file and follow the instructions there. There is no guarantee about how close this is compared with the ground truth, but this toolbox is a convention used by most works. The scheme is based on [this work](http://www.cs.huji.ac.il/~yweiss/Colorization/) as given by the website of the NYUv2 dataset.

**Comment by answerwwfthu** [comment]
@SuVanHuynh Could you please give more details about how to generate the result using tool box?

**Comment by victoriamazo** [comment]
Hi! Could you please explain how to convert exactly sparse to dense depth map? I have not found NYU tool box in python.

**Comment by LuizGuzzo** [comment]
> Hi! Could you please explain how to convert exactly sparse to dense depth map? I have not found NYU tool box in python.

It's been a while since your question, but ialhashim made a python code that converts KITTI's sparse to dense.
https://gist.github.com/ialhashim/be6235489a9c43c6d240e8331836586a#file-fill_depth_colorization-py
